tests/system/model_monitoring/test_app.py

```diff
@@ -99,6 +99,10 @@ class _V3IORecordsChecker:
 
     @classmethod
     def custom_setup(cls, project_name: str) -> None:
+        # By default, the TSDB connection is based on V3IO TSDB
+        # To use TDEngine, set the `TSDB_CONNECTION` environment variable to the TDEngine connection string,
+        # e.g. "taosws://user:password@host:port"
```

Comment:
That's good once the DB is available.
What do you think about docs for setting this DB up?

Reply:
amm we can add it to internal docs (such as `dev-docs`) but I don't think we should add it to MLRun docs. TDEngine has its own docs on how to deploy it with different approaches (docker, k8s, ...).

Reply:
Ok, internal documentation is acceptable if we provide it out of the box in CE 👍